core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -695,7 +695,8 @@ private[spark] class TaskSetManager(
   def handleSuccessfulTask(tid: Long, result: DirectTaskResult[_]): Unit = {
     val info = taskInfos(tid)
     val index = info.index
-    info.markFinished(TaskState.FINISHED)
+    // Mark task as finished. Note that finishing time here should be bigger than 0.
```

Comment:
I'm not sure what this comment is trying to say, especially since there's no check anywhere for "finishing time > 0" in the code here.

Also, grammar nit: it should be "larger than 0".